README.md

```diff
@@ -115,6 +115,51 @@ If all your [browser targets](https://guides.emberjs.com/release/configuring-emb
 
 The way you do import remains same.
 
+### Error Handling
+
+`ember-fetch` comes with utility functions for matching responses and their response status codes.
+These utilities are to assist when transitioning from [`ember-ajax`](https://github.com/ember-cli/ember-ajax#error-handling)
```

Comment:
These utils are useful for all fetch users not only migration from ember-ajax so I think this sentence is not necessary.